src/kinematics/SwerveDriveKinematics.h

```diff
@@ -0,0 +1,113 @@
+#pragma once
+
+#include "../navtypes.h"
+
+using namespace navtypes;
+
+struct swervewheelvel_t {
+	/**
+	 * Represents robot wheel velocities in robot reference frame,
+	 * in polar form (wheel speed and angle)
+	 */
+	
+	double lfVel;
+	double lfRot;
+	double rfVel;
+	double rfRot;
+	double lbVel;
+	double lbRot;
+	double rbVel;
+	double rbRot;
+};
+
+class SwerveDriveKinematics {
```

Comment:
We should have a class-level comment too, can you just put a brief description of the class?